backend/src/models/squad.js

```diff
@@ -1,5 +1,7 @@
-const Sequelize = require('sequelize');
-const sequelize = require('../util/database');
+const Sequelize = require("sequelize");
+const sequelize = require("../util/database");
+
+// TODO: UPDATE TESTS BEFORE MERGING THIS PR!
```

Comment:
Need https://github.com/mdesson/squadgoals/pull/19 to be merged before this PR. Once it's merged I can update tests and remove this.